dinky-admin/src/main/resources/log4j2.xml

```diff
@@ -42,6 +42,17 @@
         <!--当app.log文件大小到达100MB的时候，就归档一次日志文件，也就是把app.log中的那前面100MB文件取出来，放到上面<RollingFile >中的filePattern后面的路径中-->
         <SizeBasedTriggeringPolicy size="100MB"/>
```

Comment:
File size setting smaller?

Reply:
this file is a simple type log and may be very large on special conditions, recommend not to display it. It's an unimportant context for the user. if needed, can check the log file directly.